src/pfs/pfsutil/pfsutil.go

```diff
@@ -129,14 +129,27 @@ func InspectCommit(apiClient pfs.APIClient, repoName string, commitID string) (*
 }
 
 func ListCommit(apiClient pfs.APIClient, repoNames []string) ([]*pfs.CommitInfo, error) {
+	return ListCommitComplex(apiClient, repoNames, nil, false)
+}
+
+func ListCommitComplex(apiClient pfs.APIClient, repoNames []string, fromCommitIDs []string, block bool) ([]*pfs.CommitInfo, error) {
```

Comment:
To prevent fracturing this API too much let's just merge this behavior into `ListCommit` and update the other callsites.